webapp/security/models.py

```diff
@@ -50,6 +50,35 @@
     Column("cve_reference_id", Integer, ForeignKey("cve_reference.id")),
 )
 
+cve_packages = Table(
+    "cve_packages",
+    Base.metadata,
+    Column("cve_id", String, ForeignKey("cve.id")),
+    Column("cve_packages", Integer, ForeignKey("package.id")),
+)
+
+cve_package_release_status = Table(
+    "cve_package_release_status",
+    Base.metadata,
+    Column("package_id", Integer, ForeignKey("package.id")),
+    Column(
+        "package_release_status_id",
+        Integer,
+        ForeignKey("package_release_status.id"),
+    ),
+)
+
+cve_releases = Table(
+    "cve_releases",
+    Base.metadata,
+    Column("release_id", Integer, ForeignKey("release.id")),
+    Column(
+        "package_release_status_id",
+        Integer,
+        ForeignKey("package_release_status.id"),
+    ),
+)
+
 
 class CVE(Base):
```

Comment:
Is there a reason we decided to store all dates as strings?

Reply:
Updated. I initially did it because it seems the query changes the format of the date, but we want to show it in ISO format. But I guess there is a `strftime('%Y-%m-%d')`, so updated.